docs/Daos.md

```diff
@@ -0,0 +1,262 @@
+
+Most often we use schemes to perform primary functions respecting a particular DAO, especially when working with proposals.  Every scheme that works with proposals also is configured with a voting machine.
+
+**Global constraints** use configured criteria to block actions attempted by the Controller that would violate a given constraint.  An example is [TokenCapGC](api/classes/TokenCapGCWrapper) that limits the total supply of a given token to a certain maximum number.
+
+DAOs are extensible beyond the reusability of all Arc contracts: You can provide your own implementation of the controller, token, and any global constraints, voting machines and schemes you want.
+
+!!! info
+    Refer here for [more information about the architecture of DAOs](https://daostack.github.io/arc/README/#the-structure-of-a-dao).
+
+!!! info
+    Refer here for [more information about proposals](Proposals).
+    
+## Creating a new DAO
+
+When creating a DAO you can configure its name, token, founders and schemes.  For schemes you configure their parameters, permissions and their voting machine.
+
+You can call [DAO.new](api/classes/DAO#new) to create a new DAO, passing it a [configuration object](api/interfaces/NewDaoConfig).
+
+Almost everything in the configuration has a default.  The following sections describe how to supply custom configurations.
+
+!!! tip
+    Under the hood, `Dao.new` uses the [DaoCreatorWrapper](api/classes/DaoCreatorWrapper) class.
+
+### Creating a new DAO with all defaults
+
+The simplest example of how to create a new DAO uses all defaults:  No schemes nor founders:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT"
+});
+```
+
+And voilà, you have created a new DAO in the DAO stack on Ethereum.  The returned object `newDao` is an instance of [DAO](api/classes/DAO) and you are off and running.  But you won't run far without schemes and founders who can make proposals and vote on them, and for that, the founders will need some reputation.
+
+### Creating a new DAO with founders
+
+Create a new DAO with founders by including a "founders" array.  This will automatically mint tokens and reputation to each founder:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  founders: [
+    {
+      // the current user
+      address: accounts[0],
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: founder1Address,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: founder2Address,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    }
+  ]
+});
+```
+
+!!! note
+    It is not possible to add or remove founders.  In fact, there is no retained sense of who they even were .  They are simply addresses you supply when you create a DAO and to whom will immediately be minted tokens and reputation.
```

Comment:
You can always mint some reputation or tokens to others..same like adding them as founders.

Reply:
Done